thumbor/requirements.txt

```diff
@@ -1,24 +1,11 @@
 envtpl==0.6.0
-circus==0.15.0
-circus-web==1.0.0
-boto==2.48.0
-botocore==1.12.253
-dateutils==0.6.6
-numpy==1.14.3
-remotecv==2.2.2
-opencv-engine==1.0.1
-thumbor==6.7.0
+remotecv==3.0.0
+boto==2.49.0
 tornado-botocore==1.5.0
-tc-aws==6.2.10
```

Comment:
@heynemann I guess we still need some of the libraries here with thumbor 7 ? which ones? and which versions?

Reply:
The problem with urllib2 is that tc-aws is not python3 compatible. https://stackoverflow.com/questions/2792650/import-error-no-module-name-urllib2
That said, I created a new aws storage library called thumbor-aws until tc-aws gets updated. We can try that... https://github.com/thumbor/thumbor-aws

Reply:
I created a new library called thumbor-aws we can use until tc-aws gets updated to python3. https://github.com/thumbor/thumbor-aws

The problem with urllib2 is that tc-aws is not python3 compliant. https://stackoverflow.com/questions/2792650/import-error-no-module-name-urllib2

Reply:
so should we add thumbor-aws to requirements.txt ?

Reply:
also, correct me if I'm wrong, but wouldn't this require a change to the configs? e.g. if someone uses `LOADER=tc_aws.loaders.s3_loaders` they would need to change it? if so, is there a way to make thumbor-aws backwards compatible, so it accepts the "old" config directives?

Reply:
Yep, I meant as an additional lib to the docker image :) That way people who want can use it while tc-aws is not compatible.

Reply:
There is :) will work on making it more compatible later...

Reply:
Great. What's the next step here then? I'm not sure I completely follow :)

Reply:
@heynemann is all this resolved since tc_aws 7.0.2 works with Thumbor 7?

Reply:
I updated tc-aws to 7.0.2. There's a new thumbor 7.4.7 image on docker hub. Can you test it? ideally with tc-aws. I think this should solve the urllib2 issue.